packages/cells/src/widget.ts

```diff
@@ -552,6 +558,13 @@ export class Cell<T extends ICellModel = ICellModel> extends Widget {
     }
   }
 
+  /**
+   * Signal emitted when cell editor requests scrolling.
+   */
+  get editorScrollRequested(): ISignal<Cell, Cell.IEditorScrollRequest> {
```

Comment:
Based on the investigation in https://github.com/jupyterlab/jupyterlab/issues/16007#issuecomment-2003496842 it appears likely that a similar signal will be needed for stdin box to fix a related issue with cell not being scrolled to when user types into the stdin box. Maybe it is better to rename this signal to a generic `scrollRequested`?

Reply:
Change made in 438e829